import/dproto/intermediate.d

```diff
@@ -216,6 +236,10 @@ struct Field {
 	{
 		switch(fmt.spec) {
 			case 'p':
+				if(fmt.width == 3)
+				sink.formattedWrite("%s %s = %s%p; ",
```

Comment:
Indeed. I looked at some of their code in the release, some of it uses `optional` even with `syntax = "proto3"` and some doesn't...